### PR TITLE
Send booking verify mail

### PR DESCRIPTION
**Pull request by phinguyenphuoc**
image

### CODE REVIEW

**Comment by phinguyenphuoc** [comment]
ready

File: app/mailers/application_mailer.rb [hunk 1]
```diff
@@ -1,4 +1,4 @@
 class ApplicationMailer < ActionMailer::Base
-  default from: "from@example.com"
+  default from: "phinguyenphuoc@gmail.com"
```
Comment (dothidiemthao) [inline app/mailers/application_mailer.rb]: settings nhé

File: config/environments/production.rb [hunk 2]
```diff
@@ -91,4 +91,16 @@
 
   # Do not dump schema after migrations.
   config.active_record.dump_schema_after_migration = false
+
+  # Send mail via gmail
+  config.action_mailer.delivery_method = :smtp
+  # SMTP settings for gmail
+  config.action_mailer.smtp_settings = {
+   :address              => "smtp.gmail.com",
```
Comment (dothidiemthao) [inline config/environments/production.rb]: dùng format
address: "smtp.gmail.com",

**Comment by phinguyenphuoc** [comment]
ready

**Review by dothidiemthao — APPROVED** [review]

**Pull request merged**